examples/event_handling/pong_gtk_sgskip_sgskip.py

```diff
@@ -1,3 +1,9 @@
+"""
```

Comment:
good catch - ran my script one too many times :)

(I'll fix these and will overwrite the commit so that we don't have tons and tons of changes)